docs/source/releases/latest/666-final-bug-fixes-prior-to-workshop.yaml

```diff
@@ -0,0 +1,9 @@
+documentation:
+- description: |
+    Add create_plugin_registries and missing test data to README
+  title: 'Add create_plugin_registries and missing test data to README'
+  related-issue:
+    number: 666
+    url: 'https://github.com/NRLMMD-GEOIPS/geoips'
+  files:
+    modified: ['README.md']
```

Comment:
This was from a previous PR, before we updated the YAML release note formatting.